accord-core/src/test/java/accord/impl/list/ListStore.java

```diff
@@ -332,7 +332,7 @@ public void fail(Ranges ranges, Throwable failure)
                     if (!success.isEmpty())
                         fetchCompletes.add(new FetchComplete(storeId, syncPoint, success));
                 }
-                delegate.fail(ranges, new Throwable());
+                delegate.fail(ranges, new Throwable("Failed Fetch", failure));
```

Comment:
this isn't 100% related to this branch; I am forking different bug fixes from my topology change branch to keep reviews small and easier to understand... when bootstraps fail we loose the "why"...